### PR TITLE
Fix rendering of timecontribution transitions.

### PR DESCRIPTION
**Pull request by eodolphi**
Fix succeeding / failing and resetting of time contributions on various
transtions.
Fix creating and succeeding contributions on period participants.

If applicable

- [ ] Added translatable strings to `server-deployment`
- [ ] Added translations to `sever-deployment`

### CODE REVIEW

File: bluebottle/time_based/tasks.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 
 @periodic_task(
-    run_every=(crontab(minute='*/15')),
+    run_every=(crontab(minute='*')),
```
Comment (gannetson) [inline bluebottle/time_based/tasks.py:17]: Is this for testing, or do we think it's not heavy to run it this often?

File: bluebottle/time_based/tests/test_triggers.py [hunk 2]
```diff
@@ -746,58 +779,14 @@ def test_no_review_succeed_after_cancel(self):
...
-            self.participants[0].contributions.get().status,
-            'failed'
-        )
-
-    def test_reaccept(self):
-        super().test_reaccept()
-
-        self.assertEqual(
-            self.participants[0].contributions.get().status,
-            'new'
```
Comment (gannetson) [inline bluebottle/time_based/tests/test_triggers.py:800]: Don't we want these?

**Review by gannetson — CHANGES_REQUESTED** [review]

File: bluebottle/time_based/tests/test_triggers.py (hunk 2 above)
Comment (eodolphi) [reply bluebottle/time_based/tests/test_triggers.py:800]: I moved those up to the parent tests.

File: bluebottle/time_based/tasks.py (hunk 1 above)
Comment (eodolphi) [reply bluebottle/time_based/tasks.py:17]: That was for testing. I'll revert that.

**Pull request merged**